site/content/docs/developing/sidecars.en.md

```diff
@@ -38,6 +38,9 @@ sidecars:
     # Optional Docker labels to apply to this container.
     labels:
       {label key} : <label value>
+    # Optional container dependencies to apply to this container.
+    depends_on:
+      {container name}: <status>
```

Comment:
I wonder if 
```
      {container name}: <desired status>
```
or
```
      {container name}: <condition>
```
would help folks understand what the RHS want?

Reply:
Good point! Adjusted phrasing and added the field to common-svc-fields to address your other note.